dp ← Auto-switch to draft version on first edit (#27507)
directus/directus: .changeset/cms-1868-auto-switch-draft-on-edit.md

---
'@directus/app': minor
---

> Auto-switch to the draft version on the first edit when the published item has no pending draft edits. The form appears editable immediately; typing transitions to the draft version with the edit preserved. @alvarosabu

Reviewer: Let's tighten the changeset.

```suggestion
Added behavior to auto-switch to the draft version on the first edit of published item @alvarosabu
```
